Resources/Storage/StorageBase.py

```diff
@@ -221,10 +221,6 @@ def getCurrentURL( self, fileName ):
     :param self: self reference
     :param str fileName: path on storage
     """
-    if fileName.startswith( '/' ):
-      # Assume full path is given, e.g. LFN
```

Comment:
It is still correct, just that in this case the SE definition base path is used and not the locally stored CWD.